__tests__/app.test.js

```diff
@@ -194,4 +195,15 @@ describe("PATCH api/reviews/:review_id", () => {
         expect(response.body).toEqual({ msg: "Not Found" });
       });
   });
+  test.only("200: returned object also contains a comment_count property", () => {
```

Comment:
you must remove the `.only` and ensure that all tests are passing before proceeding